File: benchmarks/fib/fib.go
package main

import "fmt"

func main() {
	x := fib(35)
	fmt.Println("Go:", x)
}

func fib(n int) int {
	if n <= 1 {
		return n
	}

	return fib(n-1) + fib(n-2)
}

File: benchmarks/fib/fib.c
#include <stdio.h>

int fib(int n);

int main() {
    int n = fib(35);
    printf("C: %d\n", n);
    return 0;
}

int fib(int n) {
    if (n <= 1) {
        return n;
    }
    return fib(n - 1) + fib(n - 2);
}